spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/caffe/CaffeLoader.scala

```
@@ -259,6 +259,7 @@ class CaffeLoader[T: ClassTag](prototxtPath: String, modelPath: String,
     parameterTable.foreach {
       case (name: String, params: Table) =>
         copyParameter(name, params)
+      case _=>
```

you should throw an exception here, or mismatch will do nothing.